app/coffeescripts/models/Assignment.coffee

```diff
@@ -29,6 +30,9 @@ define [
       if (turnitin_settings = @get('turnitin_settings'))?
         @set 'turnitin_settings', new TurnitinSettings(turnitin_settings),
           silent: true
+      if (vericite_settings = @get('vericite_settings'))?
+        @set 'vericite_settings', new VeriCiteSettings(vericite_settings),
```

Comment:
Is this a redundant/second field in the database?  We'd like to see the turnitin_settings reused with something like a "provider" key.